packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -1,3 +1,5 @@
+@import "./icons";
```

Comment:
I think the current mechanism is `@use`.

Reply:
@use breaks the build. Not sure why, I'm don't know SaSS internals very well, or at all...